services/QuillLMS/client/app/bundles/PremiumHub/containers/diagnosticGrowthReports/index.tsx

```diff
@@ -78,27 +99,62 @@ export const DiagnosticGrowthReportsContainer = ({
 
   const overviewProps = {
     ...sharedProps,
-    handleSetSelectedDiagnosticId,
+    groupByValue: selectedGroupByValue,
+    handleSetSelectedDiagnosticType,
     handleTabChangeFromDataChip,
     handleSetSelectedGroupByValue
   }
 
   const skillSectionProps = {
     ...sharedProps,
-    selectedDiagnosticId,
-    selectedGroupByValue
+    groupByValue: selectedGroupByValue,
+    diagnosticTypeValue: selectedDiagnosticType,
+    handleSetSelectedDiagnosticType,
+    handleSetSelectedGroupByValue
   }
 
   const studentSectionProps = {
     ...sharedProps,
-    selectedDiagnosticId,
+    diagnosticTypeValue: selectedDiagnosticType,
+    handleSetSelectedDiagnosticType,
     passedStudentData: null,
     passedRecommendationsData: null,
     passedVisibleData: null
   }
 
+  function getFilterSelections(tab) {
+    requestPost('/admin_report_filter_selections/show', { report: `${FILTER_SELECTIONS_REPORT_BASE}${tab}` }, (selections) => {
+      if (selections) {
+        const { group_by_value, diagnostic_type_value } = selections.filter_selections
+        setSelectedGroupByValue(group_by_value)
+        setSelectedDiagnosticType(diagnostic_type_value)
+      } else {
+        setSelectedGroupByValue(groupByDropdownOptions[0])
+        setSelectedDiagnosticType(diagnosticTypeDropdownOptions[0])
+      }
+    })
+  }
+
+  function saveFilterSelections(tab, successCallback = () => { }) {
```

Comment:
NBD but I might define the default `successCallback` as its own function and then call the variable name here instead of putting the whole function definition in the parameter just to keep things clean.